GG-29772

akorensh opened the pull request.

akorensh committed:
GG-29772 Change JMX SQLGlobalMemoryQuota and SQLQueryMemoryQuota to numeric type

akorensh committed:
GG-29772 add getSqlQueryMemoryUsage -- how much memory sql engine occupies currently.

akorensh committed:
GG-29772 remove unnecessary logging statements.

akorensh committed:
GG-29772 remove unnecessary imports.

akorensh committed:
GG-29772 add JMX SQLGlobalMemoryQuotaBytes and SQLQueryMemoryQuotaBytes to return corresponding attributes as long values.

akorensh committed:
GG-29772 add JMX SQLGlobalMemoryQuotaBytes and SQLQueryMemoryQuotaBytes to return corresponding attributes as long values.

slukyano reviewed (changes requested):
See comments in the line comments. Also, maybe add some simple tests for this after all? See GridMBeansTest for example.

akorensh committed:
GG-29772 add JMX SQLGlobalMemoryQuotaBytes and SQLQueryMemoryQuotaBytes to return corresponding attributes as long values.

akorensh committed:
GG-29772 add test for SqlQueryMXBeanImpl JMX attributes

akorensh committed:
GG-29772 add JMX SQLGlobalMemoryQuotaBytes, SQLQueryMemoryQuotaBytes to return corresponding attributes as long values.
Change SQLGlobalMemoryQuota and SQLQueryMemoryQuota to display in mb/gb instead of showing the actual bytes.

akorensh committed:
GG-29772 add JMX SQLGlobalMemoryQuotaBytes, SQLQueryMemoryQuotaBytes to return corresponding attributes as long values.
Change SQLGlobalMemoryQuota and SQLQueryMemoryQuota to display in mb/gb instead of showing the actual bytes.
